pkg/outputs/parse.go

```diff
@@ -147,12 +149,21 @@ func parseLogLineWithChunks(text string, chunks map[string]*strings.Builder) (st
 	return text, nil
 }
 
-func Parse(chunks map[string]*strings.Builder, logText string) (*ParsedLine, error) {
+type ParseOptions struct {
+	// If non-zero, returns an error if an output line is too long.
+	MaxOutputLineLen int
```

Comment:
Let's specify the unit / how this value is used here:

```suggestion
	// OutputLineMaxBytes is the maximum number of bytes a single line of output (across
	// all output chunks) can contain. Any lines larger than this will return an error.
	//
	// Disabled if <= 0.
	OutputLineMaxBytes int
```